### ISSUE TITLE
/usr/bin/env: 'node\r': No such file or directory

### ISSUE THREAD

**Issue by vjpr** [rstacruz/pnpm#439]
### Code to reproduce the issue:

```
docker run -it nodesource/xenial:6.5.0 bash
$ npm i -g pnpm@0.42.3
$ npm i
```

### Expected behavior:

No error.

### Actual behavior:

```
/usr/bin/env: 'node\r': No such file or directory
The command '/bin/sh -c pnpm install' returned a non-zero code: 127
```

### Additional information:

 - `node -v` prints: v6.5.0
 - Windows, OS X, or Linux?: Docker Linux and macOS.

**Comment by vjpr** [comment]
Seems the issue is that there is a Windows linefeed at the end of the hash bang in the cli entry point.

`vim /usr/bin/pnpm`

```
#!/usr/bin/env node^M

"use strict";^M
```

**Comment by vjpr** [comment]
After fixing with `sed -i 's/\r//g' /usr/bin/pnpm` I am getting a message saying `loud-rejection` is missing.

**Comment by iamstarkov** [comment]
@vjpr did you have any problems with installing pnpm? it seems like your pnpm lack loud-rejection, though it is included in dependencies https://github.com/rstacruz/pnpm/blob/master/package.json#L40

**Comment by iamstarkov** [comment]
linefeed incompatibility is still a problem though

**Comment by zkochan** [comment]
OMG, is this because I have published the 0.42.3 from Windows? insane... I can republish later today from linux.

**Comment by zkochan** [comment]
Unbelievable that this is still an issue in npm https://github.com/npm/npm/issues/12371

**Comment by vjpr** [comment]
Re: `loud-rejection` error. My `sed -i 's/\r//g' /usr/bin/pnpm` command was modifying in-place and removing the symlink.

**Comment by zkochan** [comment]
I republished from Linux so the line ending issue should be fixed in v0.42.4
